README.md

```diff
@@ -36,7 +36,7 @@ config.writeSync();
 
 #### file
 
-*Required*  
+*Required*
```

Comment:
Add back to two spaces. It's probably the editor that removed them automatically.